### ISSUE TITLE
error for setup tiny_tds-1.3.0-x64-mingw32

### ISSUE THREAD

**Issue by ringrid** [rails-sqlserver/tiny_tds#361]
my env:
* windows 10 x64
* ruby 2.4.1p111 [x64-mingw32] by RubyInstaller2

describe for err:
    I use irb for test:
```
irb(main):001:0> require 'tiny_tds'
LoadError: cannot load such file -- tiny_tds/tiny_tds
        from C:/Ruby/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from C:/Ruby/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from C:/Ruby/lib/ruby/gems/2.4.0/gems/tiny_tds-1.3.0-x64-mingw32/lib/tiny_tds.rb:22:in `rescue in <top (required
)>'
        from C:/Ruby/lib/ruby/gems/2.4.0/gems/tiny_tds-1.3.0-x64-mingw32/lib/tiny_tds.rb:25:in `<top (required)>'
        from C:/Ruby/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:133:in `require'
        from C:/Ruby/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:133:in `rescue in require'
        from C:/Ruby/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:40:in `require'
        from (irb):1
        from C:/Ruby/bin/irb.cmd:19:in `<main>'
```

I found at issue and stackoverflow，found and try setup tiny_tds with version x86，same one.

so... How can I do?

**Comment by rbacarin** [comment]
I have the same issue...

**Comment by engineersmnky** [comment]
There is no pre-compiled gem for 2.4 as of yet according to the [README](https://github.com/rails-sqlserver/tiny_tds#compiling-gems-for-windows). I am assuming you would need to compile your own but I have no understand of how to assist in doing so

**Comment by metaskills** [comment]
Docs are behind. See here: https://github.com/rails-sqlserver/tiny_tds/blob/master/Rakefile#L92

**Comment by coderjoe** [comment]
I've merged PR #368 which adds RubyInstaller2 support which seems like the cause of this issue. This will be fixed in the next release. Closing this issue as a duplicate if #365.
